test/common_nn.py

```diff
@@ -1031,6 +1032,8 @@ def __call__(self, test_case):
             out = test_case._forward_criterion(module, input, target)
             expected_out = self.reference_fn(deepcopy(input),
                                              deepcopy(target), module)
+            if isinstance(expected_out, Variable):
+                expected_out = expected_out.item()
```

Comment:
why is this necessary?

Reply:
The assertEquals overload we have handles comparisons between tensors differently from comparisons with numbers.